packages/styles/src/core/layout/toolbar/_main.scss

```diff
@@ -1,36 +1,39 @@
+$_toolbar-main__size: 13 * $u1;
+
 .as-toolbar-main {
   display: flex;
-  height: 52px;
-  background-color: var(--brand-02-color, $brand-02-color);
-
-  &--bg-brand-1 {
-    background-color: var(--brand-01-color, $brand-01-color);
-  }
-
-  &--bg-brand-2 {
-    background-color: var(--brand-02-color, $brand-02-color);
-  }
+  height: $_toolbar-main__size;
+  background-color: var(--toolbar-main-bg-color, $brand-02-color);
 
-  &--bg-brand-3 {
-    background-color: var(--brand-03-color, $brand-03-color);
+  &__toggle {
+    position: absolute;
+    z-index: 1;
   }
 
   &__item {
     display: flex;
     align-items: center;
+    justify-content: center;
     filter: invert(100%);
-    width: 48px;
-    min-width: 48px;
-    height: 48px;
-    min-height: 48px;
-    padding: 8px;
+    padding: 3 * $u1;
+    text-align: center;
     cursor: pointer;
 
     p {
-      margin-left: 8px;
+      margin-left: $u3;
```

Comment:
We need to avoid setting CSS properties to bare elements, we need to use classes for this. I changed it in my PR.

Reply:
+1 to jbot's comment